web.py

```diff
@@ -168,16 +168,34 @@ def get_pokemarkers():
     session.close()
 
     for pokemon in pokemons:
-        markers.append({
-            'id': 'pokemon-{}'.format(pokemon.id),
-            'type': 'pokemon',
-            'trash': pokemon.pokemon_id in config.TRASH_IDS,
-            'name': POKEMON_NAMES[pokemon.pokemon_id],
-            'pokemon_id': pokemon.pokemon_id,
-            'lat': pokemon.lat,
-            'lon': pokemon.lon,
-            'expires_at': pokemon.expire_timestamp,
-        })
+            content = {
+                'id': 'pokemon-{}'.format(pokemon.id),
+                'type': 'pokemon',
+                'trash': pokemon.pokemon_id in config.TRASH_IDS,
+                'name': POKEMON_NAMES[pokemon.pokemon_id],
+                'pokemon_id': pokemon.pokemon_id,
+                'lat': pokemon.lat,
+                'lon': pokemon.lon,
+                'expires_at': pokemon.expire_timestamp,
+            }
+            
+            try:
+                if pokemon.atk_iv is not None:
+                    iv = {
+                        'atk': pokemon.atk_iv,
+                        'def': pokemon.def_iv,
+                        'sta': pokemon.sta_iv,
+                        'move1': MOVES.get(pokemon.move_1, {}).get('name'),
+                        'move2': MOVES.get(pokemon.move_2, {}).get('name'),
+                    }
+                    content.update(iv)
+                
+                
+            except KeyError:
```

Comment:
you will need a `AttributeError` here not a `KeyError` as Sighting is a Objekt not an array